### PR TITLE
Fix nunit-console package name for v3.5

### PR DESCRIPTION
**Pull request by ngyukman**

### CODE REVIEW

File: src/main/groovy/com/ullink/gradle/nunit/NUnit.groovy [hunk 1]
```diff
@@ -121,7 +121,7 @@ class NUnit extends ConventionTask {
 
     String getNunitName() {
         if (isV35OrAbove) {
-            return "NUnit.${getNunitVersion()}"
+            return "NUnit.Console-${getNunitVersion()}"
```
Comment (timotei) [inline src/main/groovy/com/ullink/gradle/nunit/NUnit.groovy:124]: This is getting super weird. They are not consistent at all :(

https://github.com/nunit/nunit-console/releases

3.6.0 doesn't even have the .zip file, and has different name :(

Comment (ngyukman) [reply src/main/groovy/com/ullink/gradle/nunit/NUnit.groovy:124]: the zip file looks there, am I getting something wrong?
or they just put it back?

Comment (gluck) [reply src/main/groovy/com/ullink/gradle/nunit/NUnit.groovy:124]: They're trolling us, they put it back in the meantime :trollface: (it wasn't there before, I checked)
Good to go with this one ?

Comment (timotei) [reply src/main/groovy/com/ullink/gradle/nunit/NUnit.groovy:124]: :+1:

**Pull request merged**